Sources/XcodeGenKit/SettingsBuilder.swift

```diff
@@ -81,7 +81,7 @@ extension Project {
         buildSettings += settings.buildSettings
 
         for (configVariant, settings) in settings.configSettings {
-            let isPartialMatch = config.name.lowercased().contains(configVariant.lowercased())
+            let isPartialMatch = config.name.lowercased().split(separator: " ").contains(Substring(configVariant.lowercased()))
```

Comment:
As far as I can see this would be a breaking change, as putting `stag` for `Staging Release` would now fail, as this is now matching whole words instead of any substring.
We could change this so it finds a config in the following order:
- an exact match
- matching a word (space delimited)
- any substring

Reply:
Got your point, thank you! I'll update the fix and add extra tests to cover this scenario as soon as have time.